fast-jar creates a different directory structure not recognized by Jib

**Describe the bug**
`quarkus.package.type=fast-jar` creates a different directory structure that it's not recognized by Jib container image.

**Expected behavior**
Jib creates the container image successfully following on the package type.

**Actual behavior**
```
25007 [ERROR] Failed to execute goal io.quarkus:quarkus-maven-plugin:1.6.0.Final:build (default) on project cm-api: Failed to build quarkus application: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
25008 [ERROR] 	[error]: Build step io.quarkus.container.image.jib.deployment.JibProcessor#buildFromJar threw an exception: java.lang.RuntimeException: Unable to create container image
25008 [ERROR] 	at io.quarkus.container.image.jib.deployment.JibProcessor.containerize(JibProcessor.java:153)
25008 [ERROR] 	at io.quarkus.container.image.jib.deployment.JibProcessor.buildFromJar(JibProcessor.java:102)
25009 [ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
25009 [ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
25009 [ERROR] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
25009 [ERROR] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
25009 [ERROR] 	at io.quarkus.deployment.ExtensionLoader$2.execute(ExtensionLoader.java:932)
25009 [ERROR] 	at io.quarkus.builder.BuildContext.run(BuildContext.java:277)
25009 [ERROR] 	at org.jboss.threads.ContextClassLoaderSavingRunnable.run(ContextClassLoaderSavingRunnable.java:35)
25010 [ERROR] 	at org.jboss.threads.EnhancedQueueExecutor.safeRun(EnhancedQueueExecutor.java:2046)
25010 [ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.doRunTask(EnhancedQueueExecutor.java:1578)
25010 [ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1452)
25010 [ERROR] 	at java.base/java.lang.Thread.run(Thread.java:834)
25010 [ERROR] 	at org.jboss.threads.JBossThread.run(JBossThread.java:479)
25011 [ERROR] Caused by: java.util.concurrent.ExecutionException: java.nio.file.NoSuchFileException: /builds/devel/cm-api/target/lib
25011 [ERROR] 	at com.google.common.util.concurrent.AbstractFuture.getDoneValue(AbstractFuture.java:552)
25011 [ERROR] 	at com.google.common.util.concurrent.AbstractFuture.get(AbstractFuture.java:513)
25012 [ERROR] 	at com.google.common.util.concurrent.FluentFuture$TrustedFuture.get(FluentFuture.java:82)
25012 [ERROR] 	at com.google.cloud.tools.jib.builder.steps.PushLayerStep.call(PushLayerStep.java:79)
25012 [ERROR] 	at com.google.cloud.tools.jib.builder.steps.PushLayerStep.call(PushLayerStep.java:33)
25012 [ERROR] 	at com.google.common.util.concurrent.TrustedListenableFutureTask$TrustedFutureInterruptibleTask.runInterruptibly(TrustedListenableFutureTask.java:125)
25013 [ERROR] 	at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:57)
25013 [ERROR] 	at com.google.common.util.concurrent.TrustedListenableFutureTask.run(TrustedListenableFutureTask.java:78)
25013 [ERROR] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
25013 [ERROR] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
25013 [ERROR] 	at java.base/java.lang.Thread.run(Thread.java:834)
25013 [ERROR] Caused by: java.nio.file.NoSuchFileException: /builds/devel/cm-api/target/lib
25013 [ERROR] 	at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:92)
25013 [ERROR] 	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
25014 [ERROR] 	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:116)
25014 [ERROR] 	at java.base/sun.nio.fs.UnixFileAttributeViews$Basic.readAttributes(UnixFileAttributeViews.java:55)
25014 [ERROR] 	at java.base/sun.nio.fs.UnixFileSystemProvider.readAttributes(UnixFileSystemProvider.java:149)
25014 [ERROR] 	at java.base/sun.nio.fs.LinuxFileSystemProvider.readAttributes(LinuxFileSystemProvider.java:99)
25014 [ERROR] 	at java.base/java.nio.file.Files.readAttributes(Files.java:1763)
25014 [ERROR] 	at java.base/java.nio.file.Files.getLastModifiedTime(Files.java:2314)
25014 [ERROR] 	at com.google.cloud.tools.jib.cache.LayerEntriesSelector$LayerEntryTemplate.<init>(LayerEntriesSelector.java:77)
25015 [ERROR] 	at com.google.cloud.tools.jib.cache.LayerEntriesSelector.toSortedJsonTemplates(LayerEntriesSelector.java:153)
25015 [ERROR] 	at com.google.cloud.tools.jib.cache.LayerEntriesSelector.generateSelector(LayerEntriesSelector.java:169)
25015 [ERROR] 	at com.google.cloud.tools.jib.cache.Cache.retrieve(Cache.java:182)
25015 [ERROR] 	at com.google.cloud.tools.jib.builder.steps.BuildAndCacheApplicationLayerStep.call(BuildAndCacheApplicationLayerStep.java:102)
25015 [ERROR] 	at com.google.cloud.tools.jib.builder.steps.BuildAndCacheApplicationLayerStep.call(BuildAndCacheApplicationLayerStep.java:38)
25015 [ERROR] 	... 6 more
```

**Environment (please complete the following information):**
 - Output of `uname -a` or `ver`: Linux 5.4.0-40-generic #44-Ubuntu SMP Tue Jun 23 00:01:04 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
 - Output of `java -version`: openjdk version "14.0.1" 2020-04-14
 - GraalVM version (if different from Java): 
 - Quarkus version or git rev: 1.6.0.Final
 - Build tool (ie. output of `mvnw --version` or `gradlew --version`): 3.6.3

--- a/extensions/container-image/container-image-jib/deployment/src/main/java/io/quarkus/container/image/jib/deployment/JibProcessor.java
+++ b/extensions/container-image/container-image-jib/deployment/src/main/java/io/quarkus/container/image/jib/deployment/JibProcessor.java
@@ -250,7 +250,7 @@ public class JibProcessor {
     private JibContainerBuilder createContainerBuilderFromFastJar(JibConfig jibConfig,
             JarBuildItem sourceJarBuildItem,
             List<ContainerImageLabelBuildItem> containerImageLabels) {
-        Path componentsPath = sourceJarBuildItem.getPath().getParent().getParent();
+        Path componentsPath = sourceJarBuildItem.getPath().getParent();
 
         AbsoluteUnixPath workDirInContainer = AbsoluteUnixPath.get("/work");
 

